packages/amesos2/CMakeLists.txt

```diff
@@ -62,11 +62,6 @@ TRIBITS_ADD_OPTION_AND_DEFINE(${PACKAGE_NAME}_ENABLE_TIMERS
 #
 
 
-TRIBITS_ADD_OPTION_AND_DEFINE(${PACKAGE_NAME}_ENABLE_LAPACK
-  HAVE_${PACKAGE_NAME_UC}_LAPACK
-  "Enable the dense serial solver interface"
-  OFF )
```

Comment:
@cgcgcg double-checking here. since you remove this and set LAPACK to an optional TPL, does that mean the macro name changed? I could be confusing this with our MueLu naming convention, but wouldn't it be something like `${PACKAGE_NAME}_HAVE_LAPACK`?

Reply:
The issue was that Amesos2 didn't declare LAPACK as a dependency. So tribits didn't set up these variables. Now it does, and the variable is called `Amesos2_ENABLE_LAPACK` and sets `HAVE_AMESOS2_LAPACK`. So this shouldn't break backwards compatibility unless I'm mistaken. This is the same naming scheme as in MueLu.

Reply:
gotcha. that's what I was thinking of. that makes sense to me!